infoschema/cache.go

```diff
@@ -50,21 +52,38 @@ func (h *InfoCache) GetLatest() InfoSchema {
 func (h *InfoCache) GetByVersion(version int64) InfoSchema {
 	h.mu.RLock()
 	defer h.mu.RUnlock()
-	metrics.InfoCacheCounters.WithLabelValues("get").Inc()
+	metrics.InfoCacheCounters.WithLabelValues("get_by_version").Inc()
```

Comment:
I think it is reasonable to do `.WithLabelValues("get", "version")` after modifying the definition of `InfoCacheCounters` to two labels.

BTW, counters of `GetLatest` is not updated.